client/config/deploymentSpecificConfig.js

```diff
@@ -21,3 +21,5 @@ tieConfig.constant('SERVER_URL', null);
 tieConfig.constant('PRIMER_DIRECTORY_URL', 'docs/');
 
 tieConfig.constant('ALLOW_PRINTING', false);
+
+tieConfig.constant('PARENT_PAGE_URL_ORIGIN', null);
```

Comment:
Ah OK, then PARENT_PAGE_ORIGIN is fine too (and so is the current name).

It might be helpful to add a comment to the MDN page. (`// For the definition of an origin, please see ...`)

Reply:
Added. Left as PARENT_PAGE_URL_ORIGIN to make it more easily understood.